python/cugraph/centrality/betweenness_centrality.py

```diff
@@ -26,29 +30,44 @@ def betweenness_centrality(G, k=None, normalized=True,
     G : cuGraph.Graph
         cuGraph graph descriptor with connectivity information. The graph can
         be either directed (DiGraph) or undirected (Graph)
-    k : int, optional
-        Default is None.
+
+    k : int or list or None, optional, default=None
         If k is not None, use k node samples to estimate betweenness.  Higher
         values give better approximation
+        If k is a list, use the content of the list for estimation
+
     normalized : bool, optional
         Default is True.
         If true, the betweenness values are normalized by
         2/((n-1)(n-2)) for Graphs (undirected), and
         1 / ((n-1)(n-2)) for DiGraphs (directed graphs)
         where n is the number of nodes in G.
-    weight : cudf.Series
+
+    weight : cudf.Series, optional, default=None
         Specifies the weights to be used for each vertex.
-    endpoints : bool, optional
+
+    endpoints : bool, optional, default=False
         If true, include the endpoints in the shortest path counts
+
+    implementation : string, optional, default=None
+        if implementation is None or "default", uses native cugraph,
+        if "gunrock" uses gunrock based bc
+
     seed : optional
-        k is specified and seed is not None, use seed to initialize the random
-        number generator
+        if k is specified and seed is not None, use seed to initialize the
+        random number generator
+
+    result_dtype : np.float32 or np.float64, optional, default=np.float32
```

Comment:
I understand bc scores can be very big. Should this be np.float64 by default?

Reply:
BC score themselves can reach high values (roughly  |V|^2) but its extremely unlikely (this value should be achieved only when one node is crossed by every single shortest path).

Reply:
Ok, so fp32 is always good enough in practice?

Reply:
Moved the default value to float64 in 4bdf822 as it is less restrictive and kept the option to return float32, it also avoids some casts between the output result type and `sp_counter` and `deltas`  as they are already in double